gatling-http/src/main/scala/io/gatling/http/ahc/HttpEngine.scala

```diff
@@ -73,27 +73,33 @@ class HttpEngine(
 
       import httpComponents._
 
+      val debugEnabled = logger.underlying.isDebugEnabled
+
       if (httpProtocol.enginePart.perUserNameResolution) {
         // eager load
         val _ = defaultDnsNameResolver
       }
 
       httpProtocol.warmUpUrl match {
         case Some(url) =>
+          val connectionTimeout: Int = 1000
           val requestBuilder = new RequestBuilder().setUrl(url)
             .setHeader(Accept, "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8")
             .setHeader(AcceptLanguage, "en-US,en;q=0.5")
             .setHeader(AcceptEncoding, "gzip")
             .setHeader(Connection, KeepAlive)
             .setHeader(UserAgent, "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.8; rv:16.0) Gecko/20100101 Firefox/16.0")
-            .setRequestTimeout(1000)
+            .setRequestTimeout(connectionTimeout)
 
           httpProtocol.proxyPart.proxy.foreach(requestBuilder.setProxyServer)
 
           try {
             ahcFactory.defaultAhc.executeRequest(requestBuilder.build).get
           } catch {
-            case NonFatal(e) => logger.info(s"Couldn't execute warm up request $url", e)
+            case NonFatal(e) => if (debugEnabled)
+              logger.debug(s"Couldn't execute warm up request $url", e)
+            else
+              logger.info(s"Couldn't execute warm up request $url after {} ms", connectionTimeout)
```

Comment:
directly inject `connectionTimeout` in a placeholder in the interpolated String, just like `$url`.

Reply:
Please align log messages. The only difference should be the stacktrace in DEBUG while INFO should only have the Exception message (use Throwables.detailedMessage).

Reply:
Do we really need `detailedMessage` in the info log level output? Throwable.getMessage is already detailed enough in my view:

> 18:10:14.093 [INFO ] i.g.h.a.HttpEngine - Couldn't execute warm up request http://gatling111.io: java.net.UnknownHostException: gatling111.io: nodename nor servname provided, or not known

Reply:
Sorry, I don't get you. `Throwable.getMessage` doesn't report the class name, and can return null if the Throwable wasn't created with a message.

Reply:
Sorry for the confusion, the method names misled me into thinking they are more detailed than they really are. Throwable.getMessage has more or less the same output as Throwables.detailedMessage.

It's done now:

```
11:53:13.747 [INFO ] i.g.h.a.HttpEngine - Start warm up
11:53:13.942 [INFO ] i.g.h.a.HttpEngine - Couldn't execute warm up request http://gatling111.io: j.u.c.ExecutionException: java.net.UnknownHostException: gatling111.io: nodename nor servname provided, or not known
11:53:13.944 [INFO ] i.g.h.a.HttpEngine - Warm up done
```